package/tsup.config.ts

```diff
@@ -4,7 +4,7 @@ import { peerDependencies } from "./package.json";
 export default defineConfig((options) => {
 	const dev = !!options.watch;
 	return {
-		entry: ["src/**/*.(ts|js)", "!src/**/*.d.ts"],
+		entry: ["src/**/*.(ts|js)", "!src/internal/ambient.d.ts"],
```

Comment:
We need to check that the `.d.ts` files are moved into the correct place during build. Those are in the other PR so we don't know from just this one (unless someone tested locally already)